File: 242-valid-anagram/valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        vector<int>arr(26);
        for(char c:s)
        arr[c-'a']++;
        for(char c:t)
        arr[c-'a']--;

        for(int i:arr)
        if(i!=0)
        return 0;
        return 1;
    }
};

File: 242-valid-anagram/valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
   int[] alphabets_counter = new int[26];
        if(s.length()!=t.length()){
            return false;
        }
        for (char c : s.toCharArray())
            alphabets_counter[c-'a']++;

        for (char c : t.toCharArray()){
            if (alphabets_counter[c-'a'] == 0) return false;
            alphabets_counter[c-'a']--;
        }
        return true;
}
}